node/pkg/watchers/algorand/watcher.go

```diff
@@ -275,7 +275,7 @@ func (e *Watcher) Run(ctx context.Context) error {
 			}
 
 		case <-timer.C:
-			status, err := algodClient.Status().Do(context.Background())
+			status, err := algodClient.Status().Do(ctx)
```

Comment:
Is this change intentional? Any reason we might want to use a background context here instead?

Reply:
I think this change is good. The call should be interrupted if the parent context gets cancelled.